src/main/java/com/juniorstart/juniorstart/model/User.java

```diff
@@ -52,4 +51,7 @@
     private AuthProvider provider;
 
     private String providerId;
+    
+    @OneToMany(fetch = FetchType.LAZY, mappedBy = "offerCreator",cascade =  CascadeType.ALL)
+    private Set<JobOffer> jobOffers = new HashSet<>();
```

Comment:
Hmm, nie lepiej to wrzucić do UserProfile? Ale niech ktoś się wypowie

Reply:
+1 Wrzuciłbym to do encji UserProfile

Reply:
@Ejden dobry pomysł